packages/lambdas/src/ihe-gateway-v2-inbound-document-query.ts

```diff
@@ -50,11 +50,11 @@ export async function handler(event: APIGatewayProxyEventV2) {
 
       return buildResponse(200, xmlResponse);
     } catch (error) {
-      log(`Error processing event on ${lambdaName}: ${errorToString(error)}`);
-      return buildResponse(400, error);
+      log(`Client error on ${lambdaName}: ${errorToString(error)}`);
+      return buildResponse(400, "Schema Validation Error");
```

Comment:
There is a weird thing here where we were returning 500s instead of 400s because if the body is an Error object or another complex object, it might not be serialized correctly, and was leading to unexpected behavior.